shared/login/register/row.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React from 'react'
```

Comment:
if it's a flow file than you can make a type declaration that includes the props so that things that use `Row` will be type checked. I don't think currently the use of `Row` is type checked.